### PR TITLE
Add new Jenkins operator and o-a jobs

### PR DESCRIPTION
**Pull request by 0xmichalis**
/cc @stevekuznetsov

### CODE REVIEW

File: cluster/ci/config/prow/config.yaml [hunk 1]
```diff
@@ -299,6 +299,26 @@ presubmits:
     trigger: "((?m)^/test origin-verify,?(\\s+|$))"
 
   openshift/openshift-ansible:
+  - name: openshift/job/origin/job/build/
```
Comment (stevekuznetsov) [inline cluster/ci/config/prow/config.yaml]: Would be nice if the Jenkins logic could add in the `job/` segments itself when it's asked as the real name of the job is `openshift/origin/build`

**Comment by kargakis** [comment]
Let's merge this once we deploy https://github.com/kubernetes/test-infra/pull/6146

**Comment by kargakis** [comment]
Ok, operators are updated with the fix.

**Comment by kargakis** [comment]
https://deck-ci.svc.ci.openshift.org/?job=openshift%2Fjob%2Forigin%2Fjob%2Fbuild%2F

I removed the trailing slash

File: cluster/ci/config/prow/config.yaml (hunk 1 above)
Comment (kargakis) [reply cluster/ci/config/prow/config.yaml]: Hrm, not sure if there is a sane way to do this (how do we know that the name needs to be expanded? Just by the existence of slashes?) or whether we want to spend time fixing it at all.

**Comment by stevekuznetsov** [comment]
(I will use this when the jobs are stable, which did not end up happening last night)

**Comment by stevekuznetsov** [comment]
Will fold into my own PR, thanks @kargakis 
/close

**Pull request closed without merging**